src/theme/index.tsx

```diff
@@ -72,7 +72,7 @@ export function colors(darkMode: boolean): Colors {
     bg3Opacity4: darkMode ? '#40444F69' : '#69dcdbdc69',
     bg4: darkMode ? '#565A69' : '#CED0D9',
     bg5: darkMode ? '#6C7284' : '#888D9B',
-    bg6: darkMode ? '#243036' : '#FFFFFF',
+    bg6: '', // TODO
```

Comment:
if this is not used any more, please remove it

Reply:
I will leave it empty in order to remind someone to add `bg6` instead of `bg23`. I guarantee that no code uses `bg6` now.

Reply:
if no code uses bg6 now, please remove it

Reply:
Done.